.env

```diff
@@ -0,0 +1 @@
+VUE_APP_BACKEND_URL='https://books-training-rails.herokuapp.com/api/v1/'
```

Comment:
Add this file to `.gitignore` to prevent it from being pushed to the repository.

In the `.env` file there is usually sensitive information como tokens or api keys. Not to share for security.